apphost-framework-lookup/test.json

```diff
@@ -6,6 +6,9 @@
   "versionSpecific": false,
   "type": "bash",
   "cleanup": true,
+  "skipWhen": [
+    "ci" // portable runtime packs not published
```

Comment:
The packages are probably getting published to an development-only feed, right? Should we somehow find and add that so we can run more of these tests?

Reply:
I don't plan to look into this as part of this PR.